Source/MoreDotNet.Test/Extensions/Common/RandomExtensions/NextDateTime.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace MoreDotNet.Tests.Extensions.Common.RandomExtensions
+{
+    using System;
+    using System.Linq;
+
+    using MoreDotNet.Extensions.Common;
+
+    using Xunit;
+
+    public class NextDateTime
+    {
+        [Fact]
+        public void NextDateTime_ShouldReturnDateTime()
+        {
+            var random = new Random();
+            var result = random.NextDateTime();
+
+            Assert.IsType<DateTime>(result);
```

Comment:
Again - can you think of a scenario where the NextDateTime() method will return anything else?